tools/extract_metadata.py

```diff
@@ -259,9 +259,25 @@ def get_string_at(module, address):
   return data_to_string(data[offset:str_end])
 
 
+class Metadata:
+  imports: []
+  export: []
+  asmConsts: []
+  emJsFuncs: {}
+  emJsFuncTypes: []
+  features: []
+  globalImports: []
+  invokeFuncs: []
+  mainReadsParams: bool
+  namedGlobals: []
```

Comment:
Do I remember correctly that this makes these fields "static" members rather than instance members? If so, we might not even need to instantiate a `Metadata` down in `extract_metadata`.

Reply:
They are act as default values for instance variables.  When we do `instance.foo` below we are assigning the instance, not the class.  Here we do want the ability to make unique instances.